### PR TITLE
Nick's Submission

### PR DESCRIPTION
**Pull request by nronnei**
Thanks for the opportunity to show you what I've got! 

As I mentioned in the interview, I've never worked with AngularJS. I really enjoyed this challenge and getting to know the framework! I was very pleased to see the testing sytnax is similar to that of Mocha, which I have worked with before.

I hope you like it and look forward to hearing from you,
Nick

### CODE REVIEW

File: public/_css/style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.noselect {
+  -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+     -khtml-user-select: none; /* Konqueror HTML */
+       -moz-user-select: none; /* Firefox */
+        -ms-user-select: none; /* Internet Explorer/Edge */
+            user-select: none; /* Non-prefixed version, currently
```
Comment (aaronArinder) [inline public/_css/style.css:7]: `rIgHt jUsTiFiEd fO LyFe`

**Comment by aaronArinder** [comment]
Stephen mentioned someone recently closed a pull request to this repo; checked it out, saw your old PR. Pretty cool code!

**Comment by nronnei** [comment]
Thanks, dude :slightly_smiling_face:

**Comment by biggerfisch** [comment]
I'm still subscribed to this repo I learned. Nice @nronnei

**Comment by biggerfisch** [comment]
@nronnei guess who still still still was subscribed lmao

**Pull request closed without merging**